### PR TITLE
Fixed issue #24430

### LINKED ISSUE #24430: Links to virtual products are in Refund emails when refunded per line
**Describe the bug**
There are two ways to refund an order:
- by changing order status to Refunded
- by clicking Refund button and choosing items to refund

When using the second way for refunding virtual downloadable products, Refund email still contains Downloads section with links even though links lead to `403` page if you click on them.

**To Reproduce**
1. Create a virtual downloadable product:

https://cld.wthms.co/mlTosj 
Full Size: https://cld.wthms.co/mlTosj

2. Order it and refund it this way:

https://cld.wthms.co/JHQZcX 
Full Size: https://cld.wthms.co/JHQZcX

https://cld.wthms.co/kXu3GM 
Full Size: https://cld.wthms.co/kXu3GM

3. Check Refund email - it contains Downloads section with broken links:

https://cld.wthms.co/R2XsDK 
Full Size: https://cld.wthms.co/R2XsDK

**Expected behavior**
If I refund an order by changing the order status to Refunded, there is no Download section:

https://cld.wthms.co/XQrUDH 
Full Size: https://cld.wthms.co/XQrUDH

However, in this case, store owners have to also refund the amount through payment gateway (Stripe in my case).

The behaviour is confusing for customers and I'd expect no downloads section in Refund emails.

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
```
`
### WordPress Environment ###

WordPress address (URL): https://redacted.mystagingwebsite.com
Site address (URL): https://redacted.mystagingwebsite.com
WC Version: 3.7.0
REST API Version: ✔ 1.0.2
Log Directory Writable: ✔
WP Version: 5.2.2
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: ✔

### Server Environment ###

Server Info: nginx
PHP Version: 7.2.21
PHP Post Max Size: 100 MB
PHP Time Limit: 300
PHP Max Input Vars: 6144
cURL Version: 7.65.3
OpenSSL/1.1.0k

SUHOSIN Installed: –
MySQL Version: 5.5.5-10.1.31-MariaDB
Max Upload Size: 100 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.7.0
WC Database Prefix: wp_
Total Database Size: 13.49MB
Database Data Size: 8.84MB
Database Index Size: 4.65MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.19MB + Index: 0.17MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_adtribes_my_conversions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_advanced_notifications: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_advanced_notification_triggers: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.20MB + Index: 0.22MB + Engine InnoDB
wp_followup_coupons: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_followup_coupon_logs: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_followup_customers: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_followup_customer_carts: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_followup_customer_notes: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_followup_customer_orders: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_followup_email_excludes: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_followup_email_logs: Data: 0.05MB + Index: 0.05MB + Engine InnoDB
wp_followup_email_orders: Data: 0.14MB + Index: 0.09MB + Engine InnoDB
wp_followup_email_tracking: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_followup_followup_history: Data: 0.16MB + Index: 0.05MB + Engine InnoDB
wp_followup_order_categories: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_followup_order_items: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_followup_subscribers: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_followup_subscribers_to_lists: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_followup_subscriber_lists: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_ms_snippets: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_options: Data: 2.06MB + Index: 0.06MB + Engine InnoDB
wp_postmeta: Data: 1.50MB + Index: 1.95MB + Engine InnoDB
wp_posts: Data: 0.38MB + Index: 0.23MB + Engine InnoDB
wp_say_what_strings: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_snippets: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wcpv_commissions: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wcpv_per_product_shipping_rules: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_bookings_availability: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_bookings_availabilitymeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_booking_relationships: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_deposits_payment_plans: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_deposits_payment_plans_schedule: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_points_rewards_user_points: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_points_rewards_user_points_log: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.05MB + Index: 0.09MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_warranty_products: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_bundled_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_bundled_items: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_exported_csv_items: Data: 0.06MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_exported_xml_items: Data: 0.13MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_per_product_shipping_rules: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_table_rates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpml_mails: Data: 2.52MB + Index: 0.00MB + Engine InnoDB
wp_xyz_ips_short_code: Data: 0.02MB + Index: 0.00MB + Engine InnoDB

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (6) ###

Meks Quick Plugin Disabler: by Meks – 1.0
WooCommerce Bookings: by WooCommerce – 1.15.0 – 1.15.1 is available – Not tested with the active version of WooCommerce
WooCommerce Stripe Gateway: by WooCommerce – 4.2.3 – Not tested with the active version of WooCommerce
WooCommerce UPS Shipping: by WooCommerce – 3.2.16 – 3.2.17 is available – Not tested with the active version of WooCommerce
WooCommerce: by Automattic – 3.7.0
WP Mail Logging: by Christian Zöller – 1.9.1

### Inactive Plugins (83) ###

Akismet Anti-Spam: by Automattic – 4.1.2
Classic Editor: by WordPress Contributors – 1.5
Event Tickets: by Modern Tribe
Inc. – 4.10.1.2

Event Tickets Plus: by Modern Tribe
Inc. – 4.10.1.3

Facebook for WooCommerce: by Facebook – 1.9.12 – Not tested with the active version of WooCommerce
Follow-Up Emails: by WooCommerce – 4.8.14 – 4.8.15 is available – Not tested with the active version of WooCommerce
Gutenberg: by Gutenberg Team – 4.4.0
Homepage Control: by WooThemes – 2.0.3
Insert PHP Code Snippet: by xyzscripts.com – 1.2.6
Jetpack by WordPress.com: by Automattic – 7.6
Klarna Payments for WooCommerce: by krokedil
klarna
automattic – 1.6.4 – Not tested with the active version of WooCommerce

Loco Translate: by Tim Whitlock – 2.2.2
Product Filters for WooCommerce: by Nexter – 1.1.15 – Not tested with the active version of WooCommerce
Say What?: by Lee Willis – 1.9.1
SIP Cookie Check for WooCommerce: by ShopitPress <hello@shopitpress.com> – 1.0.2 – Not tested with the active version of WooCommerce
Storefront Blog Customiser: by WooCommerce – 1.2.5 – 1.3.0 is available
Storefront Footer Bar: by WooThemes – 1.0.4
Storefront Mega Menus: by WooCommerce – 1.6.2 – Not tested with the active version of WooCommerce
Storefront Parallax Hero: by WooCommerce – 1.5.7
Storefront Powerpack: by WooCommerce – 1.4.15 – 1.5.0 is available
Storefront Pricing Tables: by WooCommerce – 1.1.0
Storefront Reviews: by WooCommerce – 1.0.5 – 1.0.6 is available – Not tested with the active version of WooCommerce
The Events Calendar: by Modern Tribe
Inc. – 4.8.2

WooCommerce - ShipStation Integration: by WooCommerce – 4.1.28 – 4.1.29 is available – Not tested with the active version of WooCommerce
WooCommerce Accommodation Bookings: by WooCommerce – 1.1.6 – 1.1.9 is available – Not tested with the active version of WooCommerce
WooCommerce Additional Variation Images: by WooCommerce – 1.7.15 – 1.7.16 is available – Not tested with the active version of WooCommerce
WooCommerce Admin: by WooCommerce – 0.12.0 – Not tested with the active version of WooCommerce
WooCommerce Advanced Notifications: by WooCommerce – 1.2.15 – 1.2.17 is available – Not tested with the active version of WooCommerce
WooCommerce Amazon Pay Gateway: by WooCommerce – 1.10.1 – 1.10.2 is available – Not tested with the active version of WooCommerce
WooCommerce Box Office: by WooCommerce – 1.1.12 – 1.1.13 is available – Not tested with the active version of WooCommerce
WooCommerce Brands: by WooCommerce – 1.6.9 – 1.6.10 is available – Not tested with the active version of WooCommerce
WooCommerce Bulk Variations: by Lucas Stark – 1.6.5 – Not tested with the active version of WooCommerce
WooCommerce Checkout Field Editor: by WooCommerce – 1.5.21 – 1.5.23 is available – Not tested with the active version of WooCommerce
WooCommerce Composite Products: by SomewhereWarm – 3.15.2 – 4.2.0 is available – Not tested with the active version of WooCommerce
WooCommerce Currency Converter: by WooCommerce – 1.6.13 – 1.6.14 is available – Not tested with the active version of WooCommerce
WooCommerce Customer/Order CSV Export: by SkyVerge – 4.6.3 – 4.8.0 is available – Not tested with the active version of WooCommerce
WooCommerce Customer/Order XML Export Suite: by SkyVerge – 2.5.1 – 2.6.0 is available – Not tested with the active version of WooCommerce
WooCommerce Deposits: by WooCommerce – 1.4.12 – 1.4.13 is available – Not tested with the active version of WooCommerce
WooCommerce Disability VAT Exemption: by WooCommerce – 1.3.6 – 1.3.8 is available – Not tested with the active version of WooCommerce
WooCommerce Distance Rate Shipping: by Automattic – 1.0.11 – 1.0.12 is available – Not tested with the active version of WooCommerce
WooCommerce Dynamic Pricing: by Lucas Stark – 3.1.13 – 3.1.15 is available – Not tested with the active version of WooCommerce
WooCommerce Email Customizer: by WooCommerce – 1.1.10 – 1.1.11 is available – Not tested with the active version of WooCommerce
WooCommerce EU VAT Number: by WooCommerce – 2.3.13 – 2.3.14 is available – Not tested with the active version of WooCommerce
WooCommerce eWAY Payment Gateway: by WooCommerce – 3.1.20 – 3.1.22 is available – Not tested with the active version of WooCommerce
WooCommerce FedEx Shipping: by WooCommerce – 3.4.13 – 3.4.21 is available – Not tested with the active version of WooCommerce
WooCommerce Google Analytics Integration: by WooCommerce – 1.4.10 – Not tested with the active version of WooCommerce
WooCommerce Gutenberg Products Block: by Automattic – 1.1.2 – Not tested with the active version of WooCommerce
WooCommerce Measurement Price Calculator: by SkyVerge – 3.14.2 – 3.15.0 is available – Not tested with the active version of WooCommerce
WooCommerce Memberships: by SkyVerge – 1.12.4 – 1.14.0 is available – Not tested with the active version of WooCommerce
WooCommerce Min/Max Quantities: by WooCommerce – 2.4.6 – 2.4.10 is available – Not tested with the active version of WooCommerce
WooCommerce Multi-currency: by TIV.NET INC – 1.16.0 – 1.17.1 is available – Not tested with the active version of WooCommerce
WooCommerce PayFast Gateway: by WooCommerce – 1.4.12 – 1.4.13 is available – Not tested with the active version of WooCommerce
WooCommerce PayPal Checkout Gateway: by WooCommerce – 1.6.16 – 1.6.17 is available – Not tested with the active version of WooCommerce
WooCommerce PayPal Powered by Braintree Gateway: by WooCommerce – 2.2.4 – 2.2.6 is available – Not tested with the active version of WooCommerce
WooCommerce PDF Invoices: by Andrew Benbow – 4.3.4 – 4.4.2 is available – Not tested with the active version of WooCommerce
WooCommerce Photography: by WooCommerce – 1.0.17 – 1.0.18 is available – Not tested with the active version of WooCommerce
WooCommerce Points and Rewards: by WooCommerce – 1.6.22 – 1.6.23 is available – Not tested with the active version of WooCommerce
WooCommerce Pre-Orders: by WooCommerce – 1.5.12 – 1.5.17 is available – Not tested with the active version of WooCommerce
WooCommerce Product Add-ons: by WooCommerce – 3.0.11 – 3.0.14 is available – Not tested with the active version of WooCommerce
WooCommerce Product Bundles: by SomewhereWarm – 5.10.2 – 5.12.0 is available – Not tested with the active version of WooCommerce
WooCommerce Product Bundles - Bulk Discounts: by SomewhereWarm – 1.1.0 – Not tested with the active version of WooCommerce
WooCommerce Product Bundles - Min/Max Items: by SomewhereWarm – 1.3.5 – Not tested with the active version of WooCommerce
WooCommerce Product CSV Import Suite: by WooCommerce – 1.10.20 – 1.10.24 is available – Not tested with the active version of WooCommerce
WooCommerce Product Enquiry Form: by WooCommerce – 1.2.7 – 1.2.9 is available – Not tested with the active version of WooCommerce
WooCommerce Product Vendors: by WooCommerce – 2.1.18 – 2.1.19 is available – Not tested with the active version of WooCommerce
WooCommerce Royal Mail: by WooCommerce – 2.5.16 – 2.5.17 is available – Not tested with the active version of WooCommerce
WooCommerce Services: by Automattic – 1.20.0 – 1.21.1 is available – Not tested with the active version of WooCommerce
WooCommerce Shipment Tracking: by WooCommerce – 1.6.14 – 1.6.15 is available – Not tested with the active version of WooCommerce
WooCommerce Shipping Per Product v2: by WooCommerce – 2.2.15 – 2.3.0 is available – Not tested with the active version of WooCommerce
WooCommerce Ship to Multiple Addresses: by WooCommerce – 3.6.8 – 3.6.9 is available – Not tested with the active version of WooCommerce
WooCommerce Square: by WooCommerce – 1.0.37 – 1.0.38 is available – Not tested with the active version of WooCommerce
WooCommerce Store Credit: by WooCommerce – 2.4.0 – 2.4.6 is available – Not tested with the active version of WooCommerce
WooCommerce Subscribe All the Things: by Prospress Inc. – 2.1.0-beta.2 – Not tested with the active version of WooCommerce
WooCommerce Subscriptions: by Automattic – 2.5.7 – Not tested with the active version of WooCommerce
WooCommerce Subscriptions Gifting: by Prospress Inc. – 2.0.2 – 2.0.4 is available – Not tested with the active version of WooCommerce
WooCommerce Table Rate Shipping: by WooCommerce – 3.0.15 – Not tested with the active version of WooCommerce
WooCommerce USPS Shipping: by WooCommerce – 4.4.21 – 4.4.28 is available – Not tested with the active version of WooCommerce
WooCommerce Variation Swatches and Photos: by Lucas Stark – 3.0.12 – Not tested with the active version of WooCommerce
WooCommerce Warranty Requests: by WooCommerce – 1.8.13 – 1.9.1 is available – Not tested with the active version of WooCommerce
Woody ad snippets (PHP snippets | Insert PHP): by Will Bontrager Software
LLC <will@willmaster.com>
Webcraftic <wordpress.webraftic@gmail.com> – 2.2.3

WooSlider: by WooThemes – 2.4.2
WooSlider - WooCommerce Products Slideshow: by WooThemes – 1.0.12 – 1.0.14 is available – Not tested with the active version of WooCommerce
WP Crontrol: by John Blackbourn & crontributors – 1.7.1

### Dropin Plugins (2) ###

advanced-cache.php: advanced-cache.php
object-cache.php: Memcached

### Settings ###

API Enabled: –
Force SSL: –
Currency: CAD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: accommodation-booking (accommodation-booking)
booking (booking)
bundle (bundle)
composite (composite)
external (external)
grouped (grouped)
photography (photography)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔

### WC Pages ###

Shop base: #4 - /shop/
Cart: #5 - /cart/
Checkout: #6 - /checkout/
My account: #7 - /my-account/
Terms and conditions: #512 - /wooslider-page/

### Theme ###

Name: Storefront
Version: 2.5.1
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Action Scheduler ###

Complete: 0
Oldest: 2019-07-19 11:51:49 +0000
Newest: 2019-08-19 10:13:51 +0000

Pending: 6
Oldest: 2019-08-19 11:13:51 +0000
Newest: 2019-08-24 08:27:24 +0000

Canceled: 0
Oldest: –
Newest: –

In-progress: 0
Oldest: –
Newest: –

Failed: 3
Oldest: 2019-06-15 16:33:17 +0000
Newest: 2019-07-01 15:54:54 +0000

`
```
</details>

### PR DESCRIPTION
**Pull request by itzmekhokan**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes #24430  .

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

**Review by claudiosanches — DISMISSED** [review]

File: includes/class-wc-order.php [hunk 1]
```diff
@@ -1414,6 +1414,12 @@ public function get_downloadable_items() {
 				continue;
 			}
 
+			// check item refunds
```
[the remark this answers is not in the document]
Comment (itzmekhokan) [reply includes/class-wc-order.php]: @claudiosanches I updated pull request with follow WordPress coding standards.

Comment (claudiosanches) [inline includes/class-wc-order.php]: Just need to fix coding standards for this comment, need to start with capital letter and end with period.
Like `// Check item refunds.`
Thanks, just this minor issue to check.

Comment (itzmekhokan) [reply includes/class-wc-order.php]: @claudiosanches fixed coding standards for this comment.

**Review by kloon — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**